### PR TITLE
PRL-3954:Added new case state for the court not onboarded

### PR DESCRIPTION
**Pull request by khurshedk**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by AlokDatta** [comment]
can we send out the email by publishing an event on event submitted callback please.

**Comment by khurshedk** [comment]
updated code to send email by publishing an event on event submitted callback.

File: src/main/java/uk/gov/hmcts/reform/prl/handlers/TransferToAnotherCourtEventHandler.java [hunk 1]
```diff
@@ -44,6 +58,22 @@ public void notifyAllParties(final TransferToAnotherCourtEvent event) {
...
+
+    private void sendTransferToAnotherCourtEmail(String authorization,CaseData caseData) {
+        try {
+            sendgridService.sendTransferCourtEmailWithAttachments(authorization,
+                                                                  getEmailProps(caseData.getApplicantCaseName(),
+                                                                                String.valueOf(caseData.getId()),
+                                                                                caseData.getIssueDate()),
+                                                                  caseData.getCourtEmailAddress(), getAllCaseDocuments(caseData));
+        } catch (IOException e) {
+            log.error("Failed to send Email to {}", caseData.getCourtEmailAddress());
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/handlers/TransferToAnotherCourtEventHandler.java:75]: do not print email. print it like failed to send email for this case. this email address anyway we are storing in DB.

**Comment by khurshedk** [comment]
closing this as no longer required changes already merged in other branch

**Pull request closed without merging**